installer/generate_secrets.py

```diff
@@ -352,12 +353,15 @@ def _vo_cutouts(self):
                 f"Invalid vo-cutouts cloudsql value {use_cloudsql}"
             )
 
-        aws = self.secrets["butler-secret"]["aws-credentials.ini"]
-        self._set("vo-cutouts", "aws-credentials", aws)
-        google = self.secrets["butler-secret"]["butler-gcs-idf-creds.json"]
-        self._set("vo-cutouts", "google-credentials", google)
-        postgres = self.secrets["butler-secret"]["postgres-credentials.txt"]
-        self._set("vo-cutouts", "postgres-credentials", postgres)
+        try:
```

Comment:
These and similar changes to the installer need to be removed before we can merge, since these potentially break or hide errors for other environments. If behavior needs to be conditional on the USDF, that needs to be handled some other way based on the environment name or some other conditional.

Reply:
Thanks for feedback, the installer changes came in while attempting to revert to a previous commit. I have a new branch that is rebased on lsst main. I'll open a PR with that branch and add the changes to the values files below.